python_transport/wirepas_gateway/protocol/mqtt_wrapper.py

```diff
@@ -296,7 +296,16 @@ def publish(self, topic, payload, qos=1, retain=False) -> None:
         self._publish_queue.put((topic, payload, qos, retain))
         self._publish_monitor.on_publish_request()
 
-    def subscribe(self, topic, cb, qos=2) -> None:
+    def subscribe(self, topic, cb, qos=1) -> None:
```

Comment:
I would remove the qos option everywhere.
qos=2 cannot be used anymore and qos=0, we are not using it. So I would just use qos=1